website/test-integration/examples/keyboard-focus.test.tsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { DayPicker, DayPickerProps } from 'react-day-picker';
+import {
+  getDayButton,
+  getFocusedElement,
+  getNextButton,
+  getPrevButton,
+  pressArrowDown,
+  pressEnter,
+  pressShiftTab,
+  pressTab
+} from 'react-day-picker/test/po';
+import { freezeBeforeAll } from 'react-day-picker/test/utils';
+
+import { render } from '@testing-library/react';
+import { addDays, addMonths, startOfMonth } from 'date-fns';
+
+const yesterday = new Date(2022, 5, 9);
+const today = new Date(2022, 5, 10);
+const tomorrow = new Date(2022, 5, 11);
+freezeBeforeAll(today);
+
+function renderDayPicker({ mode = 'single', ...props }: DayPickerProps) {
+  render(<DayPicker mode={mode} {...props} />);
+}
+
+function tabToDayGrid() {
+  pressTab();
+  pressTab();
+  pressTab();
+}
+
+describe.each(['ltr', 'rtl'])('when text direction is %s', (dir: string) => {
+  describe('pressing tab once', () => {
+    beforeEach(() => {
+      renderDayPicker({ dir });
+      pressTab();
+    });
+    test('should focus on the previous month button', () => {
+      expect(getPrevButton()).toHaveFocus();
+    });
+  });
+  describe('pressing tab twice', () => {
+    beforeEach(() => {
+      renderDayPicker({ dir });
+      pressTab();
+      pressTab();
+    });
+    test('should focus on the next month button', () => {
+      expect(getNextButton()).toHaveFocus();
+    });
+  });
```

Comment:
NIT: We can also remove one rendering and just continue tabbing:

when tabbing once … when tabbing again… 

```suggestion
    test('should focus on the previous month button', () => {
      expect(getPrevButton()).toHaveFocus();
    });
    describe('pressing tab again', () => {
      beforeEach(() => {
        pressTab();
      });
    test('should focus on the next month button', () => {
      expect(getNextButton()).toHaveFocus();
    });
  });
```